src/test/java/tests/BaseTestClass.java

```diff
@@ -241,6 +242,18 @@ SubmitResponseBean submitForm(String requestPath, String sessionId) throws Excep
                 SubmitResponseBean.class);
     }
 
+    SubmitResponseBean submitFormAnswers(Map<String, Object> answers, String sessionId) throws Exception {
+        SubmitRequestBean submitRequestBean = mapper.readValue
+                (FileUtils.getFile(this.getClass(), "requests/submit/submit_request.json"), SubmitRequestBean.class);
```

Comment:
The other content (besides session ID and answers) actually don't matter, that was just a placeholder. Changing to reflect that.